etc/skate-style/palette.mss

```diff
@@ -191,6 +193,12 @@ Map { font-directory: url(./fonts); }
 @ferry_text:        #566b82;
 @ferry_halo:        rgba(191,213,238,0.66);
 
+@library_text:     #6b541f;
+@fire_station_text: #B02A37;
+@police_text:      #165C96;
+@school_text:      #103f4a;
+@hospital_text:    #6d0937;
```

Comment:
**nitpick:** it looks like there's some inconsistency here in capitalization of hex values.